Demo/Common/IntegrationExamples/Protocols/InitialDataFlowProtocol.swift

```diff
@@ -16,14 +16,6 @@ internal protocol InitialDataFlowProtocol: AnyObject {
 
 extension InitialDataFlowProtocol {
 
-    internal var context: AdyenContext {
-        var analyticsConfiguration = AnalyticsConfiguration()
-        analyticsConfiguration.isEnabled = ConfigurationConstants.current.analyticsSettings.isEnabled
-        return AdyenContext(apiContext: ConfigurationConstants.apiContext,
-                            payment: ConfigurationConstants.current.payment,
-                            analyticsConfiguration: analyticsConfiguration)
-    }
-
```

Comment:
We have copied this 4 times. 

For `apiClient` we have  
```swift
internal lazy var apiClient = ApiClientHelper.generateApiClient()
```

Maybe we can have `AdyenContextHelper` and keep code there ?

Reply:
It could also be a function on the `InitialDataFlowProtocol` maybe that generates it and then locally "stores" it in the Example 🤔

Reply:
Fair 🤔